test/fromJSON.js

```diff
@@ -77,8 +77,18 @@ describe('fromJSON', function () {
   it('should handle struct', function () {
     var MyType = t.struct({
       name: t.String,
-      birthDate: MyDate
-    }, 'MyType');
+      surname: t.String,
```

Comment:
could you please add a new specific test instead of modifying an existing one?

Reply:
sure